kuzzleio/kourou
# User mapping import export

## What does this PR do?

add `user:exportMapping` and `user:importMapping` commands using `security.getUserMapping` and `security.updateUserMapping`

### How should this be manually tested?
  - Step 1 : `kourou instance:spawn`
  - Step 2 : `kourou user:exportMapping`
  - Step 3 :  check your `user/userMapping.json` file and update it by adding some fields next to `profileIds` property.
  - Step 4 :  `kourou user:importMapping user/userMapping.json` 
  - Step 5 : Check on next-console.kuzzle.io your user mapping in security tab
 

### Other changes
/

### Boyscout
/
